packages/material/scss/treelist/_layout.scss

```diff
@@ -1 +1,17 @@
 @import "~@progress/kendo-theme-default/scss/treelist/_layout.scss";
+
+@include exports("treelist/layout/material") {
+    .k-treelist {
+        tr.k-footer-template td {
+            border-width: 1px 0;
```

Comment:
This style is causing a double border - 

![image](https://user-images.githubusercontent.com/17339722/85393448-b2830500-b555-11ea-82c1-cc69834d2616.png)

Reply:
According to the PSD file, there are borders: 
https://github.com/telerik/kendo-ux/blob/master/themes/default/PSD/treelist.psd